src/components/ForkInfo.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <b-card v-if="graphData" title="Fork Statistics">
+        <b-card-text v-if="currentLongestFork != null" class="fork-info" style="font-size: 1.25em">
+            Current Longest Fork: {{currentLongestFork}}
+        </b-card-text>
+        <generic-graph :chart-data="chartData" :custom-options="customOptions"></generic-graph>
+    </b-card>
+</template>
+
+<script>
+    import {last} from 'lodash';
+    import GenericGraph from './GenericGraph'
+    import Helper from '../js/Helper'
+    import {BCard, BCardText} from 'bootstrap-vue/es/components'
+
+    export default {
+        name: "ForkInfo",
+        components: {
+            GenericGraph, BCard, BCardText
+        },
+        props: {
+            graphData: Array
+        },
+        computed: {
+            currentLongestFork: {
+                get() {
+                    if (this.graphData && this.graphData.length > 0)
+                        return last(this.graphData)['value']['numForks'];
```

Comment:
`currentLongestFork` always return the number of forks instead of the tail block height of the longest fork. Was it a mistake?

Reply:
Yes, thanks for catching it.